javascript/packages/linter/src/rules/rule-utils.ts

```diff
@@ -35,13 +36,19 @@ export abstract class BaseRuleVisitor extends Visitor {
       severity
     }
   }
+  /**
+   * Visits child nodes and records the node type in the visited array
+   */
+  visitChildNodes(node: Node): void {
+    super.visitChildNodes(node)
+  }
```

Comment:
This might be a left-over from refactoring, but I don't think we need to keep this one here.
```suggestion
```